src/components/LoginForm.jsx

```diff
@@ -66,6 +67,7 @@ function LoginForm() {
                 </Form>
                 )}
             </Formik>
+                <Button href='/register' startIcon={<PersonAddAltIcon />} sx={{ color: '#DB5752', display: 'flex', mt: 3, fontSize: 18, fontWeight: 'bold', fontFamily: 'Signika' }}>CREAR CUENTA</Button>
```

Comment:
Idem acá. En vez de `<Button href='/register' ...>` debería ser `<Button component={Link} to='/register' ...>`